docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -74,8 +76,8 @@ to the coordinator's ``config.properties`` file:
     http-server.authentication.oauth2.client-id=CLIENT_ID
     http-server.authentication.oauth2.client-secret=CLIENT_SECRET
 
-In order to enable OAuth 2.0 authentication for the Web UI, the following
-properties need to be added:
+To enable OAuth 2.0 authentication for the Web UI, the following
+property needs to be added:
```

Comment:
must be added

Reply:
okay I had just removed the superfluous "in order to"  but this suggested edit is another improvement